packages/manager/apps/container/src/container/legacy/server-sidebar/universe/useServiceLoader.ts

```diff
@@ -34,5 +35,20 @@ export default function useServiceLoader(appId: string) {
           return a.label?.localeCompare(b.label);
         });
     },
+    async loadServicesV2({applicationId, path}: {
+      applicationId: string;
+      path: string;
+    }): Promise<SidebarMenuItem[]> {
+      const response = await apiClient.v2.get(path);
+      return response.data.map((service: any) => ({
+        id: `service-${service.id}`,
+        label: service.currentState.displayName || service.id,
+        href: navigation.getURL(applicationId || appId, `#/${service.id}`),
+        serviceName: service.currentState.displayName || service.id,
+      }))
+      .sort((a: SidebarMenuItem, b: SidebarMenuItem) => {
+        return a.label?.localeCompare(b.label);
+      });
+    }
```

Comment:
Normally the all load service it's provided by 2api /service. Here we call directly apiv2. It's really wanted ?

Reply:
Do we have any benefit for continuing to have this logic in a 2api ?
This is the same endpoint we call in the vrack-services uapp.

Reply:
@chipp972 entire sidebar is loaded from the `/service` 2API. So, please stick to using the same for vrack-services as well.

Reply:
@anooparveti  It is not usable because there is not the config for v2 api
I will have to modify a lot of things on the `/service` 2api which is more risky than just using our api library on the manager to make the call.

Do we want to keep using a 2api endpoint to make those calls ?
What are the benefits compared to calling the endpoint directly in the manager (which we already do in the apps btw) ?